libs/portals/admin/core/src/lib/messages.ts

```diff
@@ -0,0 +1,24 @@
+import { defineMessages } from 'react-intl'
+
+export const m = defineMessages({
+  dashboard: {
+    id: 'portals:dashboard',
```

Comment:
In the dashboard PR I set the namespace to `admin-portal:key` since it is admin core module

Reply:
Absolutlety, fixed.